### ISSUE TITLE
Add Docker Support to enhance developer experience

### ISSUE THREAD

**Issue by asr2003** [ruxailab/web-eye-tracker-front#38]
**Title:**
Adding Docker Support to web-eye-tracker-front

**Description:**

**Problem:**
web-eye-tracker-front currently lacks Docker support, hindering its ease of deployment and scalability. Adding Docker support is essential to streamline the deployment process and make it more accessible to a broader audience.

**Solution:**
Implementing Docker support for web-eye-tracker-front would involve creating a Dockerfile and configuration to package the application into a Docker container. This containerization would simplify deployment, improve consistency across environments and enhance scalability.

**Approach:**
To achieve Docker support for web-eye-tracker-front, we can follow these steps:

1. Create a Dockerfile specifying the necessary dependencies and configuration.
2. Configure the application to work seamlessly within a Docker container.
3. Document the steps for building and running web-eye-tracker-front using Docker in the project's README.

**Additional Context:**
Adding Docker support aligns with modern development practices, making it easier for developers to set up and run web-eye-tracker-front in various environments. Docker containers provide isolation, portability, and reproducibility, contributing to a more robust and user-friendly development and deployment experience.

**Comment by asr2003** [comment]
@hvini @KarinePistili I would like to work on this issue

**Comment by Jaishree2310** [comment]
@asr2003  I have a good experience with this issue, can you please assign this to me?

**Comment by asr2003** [comment]
@KarinePistili Any insights on it?

**Comment by asr2003** [comment]
@KarinePistili

**Comment by hvini** [comment]
Hello @asr2003,

we never considered using docker on the front app cause we already have the environment properly configured, but for those who don't have it could be a good idea. your steps seems good, so, i will assign to you and then you can open a PR when you finish

**Comment by KarinePistili** [comment]
Hello @asr2003 will you able to work on this?

**Comment by asr2003** [comment]
Yes @KarinePistili. I have exams right now and will be submit a PR on this weekend :)

**Comment by KarinePistili** [comment]
Great! We will wait for you then ;)
